### ISSUE TITLE
initializing error in 3bot deployer 

### ISSUE THREAD

**Issue by Pishoy** [threefoldtech/js-sdk#2266]
![image](https://user-images.githubusercontent.com/21987477/104839776-0e372600-58cc-11eb-86e0-bc02bee8abab.png)

- nothing on logs 
```
➜  ~ redis-cli -h deploy3bot.grid.tf -p 6378
deploy3bot.grid.tf:6378> SUBSCRIBE bishoy-threebot-bishoo3bot-stdout bishoy-threebot-bishoo3bot-stderr
Reading messages... (press Ctrl-C to quit)
1) "subscribe"
2) "bishoy-threebot-bishoo3bot-stdout"
3) (integer) 1
1) "subscribe"
2) "bishoy-threebot-bishoo3bot-stderr"
3) (integer) 2

```

**Comment by waleedhammam** [comment]
related logs

```
(.venv) ➜  js-sdk git:(development_vdc) redis-cli -h deploy3bot.grid.tf -p 6378       
deploy3bot.grid.tf:6378> SUBSCRIBE waleedhammam-threebot-sunday77-stdout waleedhammam-threebot-sunday77-stderr
Reading messages... (press Ctrl-C to quit)
1) "subscribe"
2) "waleedhammam-threebot-sunday77-stdout"
3) (integer) 1
1) "subscribe"
2) "waleedhammam-threebot-sunday77-stderr"
3) (integer) 2
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "zinit: INFO (zinit::app) starting init\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "[*] Enabling ssh access ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "ssh-keygen: generating new host keys: RSA DSA ECDSA ED25519 \n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) " * Restarting OpenBSD Secure Shell server sshd\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "   ...done.\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "[*] Switching to the correct version (development) ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "Fetching origin\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "HEAD is now at 80d7f2a0 sanity check for misconfigured gateways (#2265)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "[*] Starting threebot in background ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:17.265 | INFO     | __main__:main:32 - Generating guest identity ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:19.875 | WARNING  | __main__:main:80 - can not create user with name:waleedhammam_sunday77, error={\"error\": \"3Bot name or password is incorrect\"}: Reinitalizing backup for new user\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:29.354 | INFO     | __main__:main:91 - Taking backup ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:38.148 | INFO     | __main__:main:97 - Initalizing main wallet ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:39.389 | ERROR    | __main__:main:103 - Failed to create wallet, secret wasn't passed correctly: Invalid Ed25519 Secret Seed: None...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:39.391 | INFO     | __main__:main:105 - Starting threebot ...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:41.323 | INFO     | jumpscale.core.logging.logging:register:141 - Logging from 'jumpscale.servers.gedis' is now bound to 'gedis' app\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "config_backup_1 159.65.196.75\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "config_backup_2 174.138.9.246\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"failed to connect to local application: dial tcp 127.0.0.1:443: connect: connection refused\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"failed to connect to local application: dial tcp 127.0.0.1:443: connect: connection refused\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:45.079 | INFO     | jumpscale.core.logging.logging:register:141 - Logging from 'jumpscale.servers.gedis' is now bound to 'gedis' app\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:45.228 | WARNING  | jumpscale.entry_points.threebot:create_wallets_if_not_exists:269 - skipping wallets creation, identity isn't configured yet\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"failed to connect to local application: dial tcp 127.0.0.1:443: connect: connection refused\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"failed to connect to local application: dial tcp 127.0.0.1:443: connect: connection refused\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:52.866 | INFO     | jumpscale.servers.gedis.server:start:211 - Gedis server is started at 127.0.0.1:16000...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:52.874 | INFO     | jumpscale.core.logging.logging:register:141 - Logging from 'jumpscale.servers.threebot.threebot' is now bound to 'threebot_default' app\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:22:52.875 | INFO     | jumpscale.servers.threebot.threebot:start:797 - Configuring package auth\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stdout"
3) "LISTEN    0         256              127.0.0.1:16000            0.0.0.0:*        users:((\"jsng\",pid=662,fd=15))                                                 \n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"failed to connect to local application: dial tcp 127.0.0.1:443: connect: connection refused\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"failed to connect to local application: dial tcp 127.0.0.1:443: connect: connection refused\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:22PM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021/01/17 12:23:11 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:40594->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021/01/17 12:23:17 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:54602->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021/01/17 12:23:23 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:44492->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021/01/17 12:23:29 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:44498->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:44498->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:44498->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.460 | ERROR    | jumpscale.sals.nginx.nginx:generate_certificates:300 - Generating certificate failed IMPORTANT NOTES:\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) " - Your account credentials have been saved in your Certbot\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "   configuration directory at /root/sandbox/cfg/certbot. You should\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "   make a secure backup of this folder now. This configuration\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "   directory will also contain certificates and private keys obtained\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "   by Certbot so making regular backups of this folder is ideal.\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Saving debug log to /root/sandbox/var/log/certbot/letsencrypt.log\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Plugins selected: Authenticator nginx, Installer nginx\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Obtaining a new certificate\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Performing the following challenges:\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "http-01 challenge for sunday77.3botmain.grid.tf\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Waiting for verification...\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Cleaning up challenges\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "An unexpected error occurred:\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='ca1.grid.tf', port=443): Read timed out. (read timeout=45)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "Please see the logfiles in /root/sandbox/var/log/certbot for more details.\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.499 | INFO     | jumpscale.servers.gedis.server:_on_connection:239 - New connection from ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.501 | INFO     | jumpscale.servers.gedis.server:_on_connection:269 - Executing method list_actors from actor core to client ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.503 | INFO     | jumpscale.servers.gedis.server:_on_connection:269 - Executing method info from actor core to client ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.507 | INFO     | jumpscale.servers.gedis.server:_on_connection:269 - Executing method info from actor system to client ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.509 | INFO     | jumpscale.servers.gedis.server:_on_connection:269 - Executing method list_actors from actor core to client ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.511 | INFO     | jumpscale.servers.gedis.server:_on_connection:269 - Executing method info from actor core to client ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.515 | INFO     | jumpscale.servers.gedis.server:_on_connection:269 - Executing method info from actor system to client ('127.0.0.1', 43046)\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "2021-01-17 12:23:44.584 | INFO     | jumpscale.servers.threebot.threebot:start:797 - Configuring package chatflows\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:23PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "waleedhammam-threebot-sunday77-stderr"
3) "\x1b[90m12:24PM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
```

**Comment by ranatrk** [comment]
looking into logs

**Comment by ranatrk** [comment]
Blocked on not reaching `ca1.grid.tf` as shown in above logs in the following part from redis logs:
```
3) "requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='ca1.grid.tf', port=443): Read timed out. (read timeout=45)\n"
```

```
ping ca1.grid.tf
PING ca1.grid.tf (174.138.6.79) 56(84) bytes of data.
^C
--- ca1.grid.tf ping statistics ---
1098 packets transmitted, 0 received, 100% packet loss, time 1123336ms

```

**Comment by waleedhammam** [comment]
works ok now on mainnet
![Screenshot from 2021-01-17 16-09-35](https://user-images.githubusercontent.com/10658229/104845484-6e36c800-58de-11eb-84e9-fce8dc391318.png)

**Comment by ranatrk** [comment]
on testnet keeps failing due to this issue https://github.com/threefoldtech/tfgateway/issues/58

**Comment by Pishoy** [comment]
3bot instance is started and working but still get this error while deploymet in deploy3bot.grid.tf
![image](https://user-images.githubusercontent.com/21987477/104886676-cddeb300-5972-11eb-9bc8-d927d4ea7205.png)

- logs  
```
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "2021-01-18 07:43:23.808 | INFO     | jumpscale.servers.threebot.threebot:start:814 - Threebot is running at http://localhost:80 and https://localhost:443\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "2021/01/18 07:44:46 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:47836->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:44AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47836->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:44AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47836->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:44AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:44AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:46AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:46AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "2021/01/18 07:47:21 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:47976->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:47AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47976->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:47AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47976->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:47AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:47AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "2021/01/18 07:49:57 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:48006->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:49AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48006->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:49AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48006->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:50AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:50AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "2021/01/18 07:52:32 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:48024->104.236.44.145:18000: read: connection reset by peer\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:52AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48024->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:52AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48024->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:52AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
1) "message"
2) "bishoy-threebot-bisho5bot-stderr"
3) "\x1b[90m7:52AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"


```

**Comment by waleedhammam** [comment]
> 3bot instance is started and working but still get this error while deploymet in deploy3bot.grid.tf
> ![image](https://user-images.githubusercontent.com/21987477/104886676-cddeb300-5972-11eb-9bc8-d927d4ea7205.png)
> 
> * logs
> 
> ```
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "2021-01-18 07:43:23.808 | INFO     | jumpscale.servers.threebot.threebot:start:814 - Threebot is running at http://localhost:80 and https://localhost:443\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "2021/01/18 07:44:46 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:47836->104.236.44.145:18000: read: connection reset by peer\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:44AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47836->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:44AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47836->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:44AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:44AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:45AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:46AM\x1b[0m \x1b[32mINF\x1b[0m incoming stream, connect to local application \x1b[36mremote add=\x1b[0m104.236.44.145:18000\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:46AM\x1b[0m \x1b[32mINF\x1b[0m start forwarding\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "2021/01/18 07:47:21 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:47976->104.236.44.145:18000: read: connection reset by peer\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:47AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47976->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:47AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:47976->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:47AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:47AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "2021/01/18 07:49:57 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:48006->104.236.44.145:18000: read: connection reset by peer\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:49AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48006->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:49AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48006->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:50AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:50AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "2021/01/18 07:52:32 [ERR] yamux: keepalive failed: read tcp 10.100.2.2:48024->104.236.44.145:18000: read: connection reset by peer\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:52AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m  \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48024->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:52AM\x1b[0m \x1b[1m\x1b[31mERR\x1b[0m\x1b[0m retry in 5s \x1b[31merror=\x1b[0m\x1b[31m\"accept connection failed: read tcp 10.100.2.2:48024->104.236.44.145:18000: read: connection reset by peer\"\x1b[0m\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:52AM\x1b[0m \x1b[32mINF\x1b[0m start handshake\n"
> 1) "message"
> 2) "bishoy-threebot-bisho5bot-stderr"
> 3) "\x1b[90m7:52AM\x1b[0m \x1b[32mINF\x1b[0m handshake done\n"
> ```

That's tracked in another issue https://github.com/threefoldtech/tfexplorer/issues/271

**Comment by Pishoy** [comment]
works now
